python/lsst/pipe/tasks/calibrate.py

```diff
@@ -129,6 +139,10 @@ def validate(self):
         pexConfig.Config.validate(self)
         if self.doPhotoCal and not self.doAstrometry:
             raise ValueError("Cannot do photometric calibration without doing astrometric matching")
+        if self.requireAstrometry and not self.doAstrometry:
+            raise ValueError("Astrometric solution required, but not activated")
```

Comment:
The two lines above should really be part of 53ae929 rather than ce2dc85, but it's a trivial point.